node/rpc/src/progress_api.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+#![allow(clippy::unused_async)]
+
+use std::sync::{
+    atomic::{self, AtomicU64},
+    Arc,
+};
+
+use forest_ipld::{WALK_SNAPSHOT_PROGRESS_DB_GC, WALK_SNAPSHOT_PROGRESS_EXPORT};
+use forest_rpc_api::progress_api::{GetProgressParams, GetProgressResult, GetProgressType};
+
+use crate::*;
+
+pub(crate) async fn get_progress(
+    Params((typ,)): Params<GetProgressParams>,
+) -> RpcResult<GetProgressResult> {
+    let tracker: &Arc<(AtomicU64, AtomicU64)> = match typ {
```

Comment:
```suggestion
    Params((type,)): Params<GetProgressParams>,
) -> RpcResult<GetProgressResult> {
    let tracker: &Arc<(AtomicU64, AtomicU64)> = match type {
```

Reply:
This does not compile, `type` is reserved keyword

Reply:
fiasco, we could escape it with `#` but perhaps it's too much effort